### PR TITLE
Add OSG sysconfig files

### PR DESCRIPTION
**Pull request by matyasselmeci**

### CODE REVIEW

File: scripts/gridftp-hdfs-standalone.in [hunk 1]
```diff
@@ -1,18 +1,8 @@
...
-fi
-
-if [ -e @sysconfdir_resolved@/gridftp-hdfs/gridftp-hdfs-local.conf ] ; then
-    . @sysconfdir_resolved@/gridftp-hdfs/gridftp-hdfs-local.conf
-fi
-
-if [ -e @sysconfdir_resolved@/sysconfig/gridftp-hdfs ] ; then
-    . @sysconfdir_resolved@/sysconfig/gridftp-hdfs
-fi
+. /usr/share/osg/sysconfig/globus-gridftp-server-plugin
```
Comment (bbockelm) [inline scripts/gridftp-hdfs-standalone.in]: Here - and in the `xinetd` directory - can you only source if this file exists?

Comment (matyasselmeci) [inline scripts/gridftp-hdfs-standalone.in]: It's going to raise an error, but the error is going to be ignored. Actually, it's been broken for a long time now -- we used to source `.../globus-gridftp-server` but the file has been called `.../globus-gridftp-server-plugin`.

Comment (bbockelm) [inline scripts/gridftp-hdfs-standalone.in]: Oh yeah - I just want to make sure that nothing gets printed out to `stderr`.

Comment (matyasselmeci) [inline scripts/gridftp-hdfs-standalone.in]: If you want, I can make it check for existence before trying to source it.

**Comment by brianhlin** [comment]
Addressing review in #12

**Pull request closed without merging**